src/figuresClasses.js

```diff
@@ -0,0 +1,80 @@
+"use strict";
+Object.defineProperty(exports, "__esModule", { value: true });
+exports.Rectangle = exports.Circle = exports.Triangle = void 0;
+exports.getInfo = getInfo;
+var Triangle = /** @class */ (function () {
+    function Triangle(color, a, b, c) {
+        this.color = color;
+        this.a = a;
+        this.b = b;
+        this.c = c;
+        this.shape = 'triangle';
+        this.area = this.getArea();
+        if (color === undefined ||
+            a === undefined ||
+            b === undefined ||
+            c === undefined) {
+            throw new Error('All values must be entered');
+        }
+        if (a <= 0 || b <= 0 || c <= 0) {
+            throw new Error('All sides values must be > 0');
+        }
+        var sides = [a, b, c];
+        sides.splice(sides.indexOf(Math.max(a, b, c), 1));
+        if (Math.max(a, b, c) >= sides[0] + sides[1]) {
+            throw new Error("sides ".concat(a, ", ").concat(b, " and ").concat(c, " can't form a triangle"));
+        }
+    }
+    Triangle.prototype.getArea = function () {
```

Comment:
[CODE STYLE] - The `getArea` function doesn't need to accept any parameters. All necessary values are already available as object fields, so you can remove the parameters and use `this.a`, `this.b`, and `this.c` directly within the method.